.github/workflows/playwright-audit.yml

```diff
@@ -0,0 +1,42 @@
+name: Audit a Playwright test
+run-name: ${{ github.actor }} is running test ${{ inputs.test-path }} ${{ inputs.repeat-each }} times
+
+on:
+  workflow_dispatch:
+    inputs:
+      repeat-each:
+        default: 5
+        type: number
+        description: The number of times to repeat the test
+      test-path:
+        required: true
+        type: string
+        description: A relative path from root directory to the test file under test (e.g. `tests/button.test.tsx:10`)
```

Comment:
comment (non-blocking): might be worth having an example that is to a `pw` test file in this description